src/containers/Tenant/Schema/SchemaViewer/__tests__/prepareData.test.ts

```diff
@@ -0,0 +1,272 @@
+                                DefaultValue: {},
+                                ColumnFamilyId: 0,
+                            },
+                            {
+                                Id: 4,
+                                Name: 'body',
+                                Type: 'Utf8',
+                                TypeId: 4608,
+                                NotNull: false,
+                                StorageId: '',
+                                DefaultValue: {},
+                                ColumnFamilyId: 0,
+                            },
+                        ],
+                        KeyColumnNames: ['author', 'title', 'id'],
+                        Version: '1',
+                        Options: {
+                            SchemeNeedActualization: false,
+                        },
+                        ColumnFamilies: [
+                            {
+                                Id: 0,
+                                Name: 'default',
+                            },
+                        ],
+                    },
+                    ColumnShardCount: 64,
+                    Sharding: {
+                        ColumnShards: ['72075186224107692'],
+                        HashSharding: {
+                            Function: 'HASH_FUNCTION_CONSISTENCY_64',
+                            Columns: ['id', 'author', 'title'],
+                        },
+                    },
+                    StorageConfig: {
+                        DataChannelCount: 64,
+                    },
+                },
+            },
+        };
+        const result = [
+            {
+                id: 1,
+                name: 'title',
+                keyColumnIndex: 1,
+                partitioningColumnIndex: 2,
+                type: 'Utf8',
+                notNull: true,
+            },
+            {
+                id: 2,
+                name: 'author',
+                keyColumnIndex: 0,
+                partitioningColumnIndex: 1,
+                type: 'Utf8',
+                notNull: true,
+            },
+            {
+                id: 3,
+                name: 'id',
+                keyColumnIndex: 2,
+                partitioningColumnIndex: 0,
+                type: 'Int64',
+                notNull: true,
+            },
+            {
+                id: 4,
+                name: 'body',
+                keyColumnIndex: -1,
+                partitioningColumnIndex: -1,
+                type: 'Utf8',
+                notNull: false,
+            },
+        ];
+        expect(prepareSchemaData(EPathType.EPathTypeColumnTable, data)).toEqual(result);
+    });
+    it('returns empty array if data is undefined, empty or null', () => {
+        expect(prepareSchemaData(EPathType.EPathTypeTable, {})).toEqual([]);
+        expect(prepareSchemaData(EPathType.EPathTypeTable, undefined)).toEqual([]);
+        expect(prepareSchemaData(EPathType.EPathTypeTable, null)).toEqual([]);
+    });
```

Comment:
There is no external tables in our dev clusters and there is no possibility to add them since feature flag is turned off and there is issue with flags. I didn't want to use generated response instead of real backend data, so I left external tables without tests for now